R/mmrm-package.R

```diff
@@ -8,12 +8,13 @@
 #' @useDynLib mmrm, .registration = TRUE
 #' @importFrom Rcpp evalCpp
 #' @import checkmate
-#' @importFrom lifecycle deprecated
+#' @importFrom lifecycle deprecated badge
 #' @importFrom Matrix .bdiag
 #' @importFrom methods is
-#' @importFrom stats acf
+#' @importFrom stats acf na.omit get_all_vars confint
```

Comment:
same here, we should use `stats::na.omit()` etc.

Reply:
removed